rotkehlchen/tests/unit/decoders/test_balancer.py

```diff
@@ -139,42 +140,42 @@ def test_balancer_v1_exit(ethereum_inquirer, ethereum_accounts):
             sequence_index=1,
             timestamp=timestamp,
             location=Location.ETHEREUM,
-            event_type=HistoryEventType.SPEND,
-            event_subtype=HistoryEventSubType.RETURN_WRAPPED,
-            asset=A_BPT,
-            balance=Balance(amount=FVal('0.042569019597126949')),
+            event_type=HistoryEventType.WITHDRAWAL,
+            event_subtype=HistoryEventSubType.REMOVE_ASSET,
+            asset=A_BAL,
+            balance=Balance(amount=FVal('0.744372160905819159')),
             location_label=user_address,
-            notes='Return 0.042569019597126949 BPT to a Balancer v1 pool',
+            notes='Receive 0.744372160905819159 BAL after removing liquidity from a Balancer v1 pool',  # noqa: E501
             counterparty=CPT_BALANCER_V1,
             address=string_to_evm_address('0x59A19D8c652FA0284f44113D0ff9aBa70bd46fB4'),
-            identifier=None,
-            extra_data={'withdrawal_events_num': 2},
         ), EvmEvent(
             tx_hash=tx_hash,
             sequence_index=2,
             timestamp=timestamp,
             location=Location.ETHEREUM,
             event_type=HistoryEventType.WITHDRAWAL,
             event_subtype=HistoryEventSubType.REMOVE_ASSET,
-            asset=A_BAL,
-            balance=Balance(amount=FVal('0.744372160905819159')),
+            asset=A_WETH,
+            balance=Balance(amount=FVal('0.010687148200906598')),
             location_label=user_address,
-            notes='Receive 0.744372160905819159 BAL after removing liquidity from a Balancer v1 pool',  # noqa: E501
+            notes='Receive 0.010687148200906598 WETH after removing liquidity from a Balancer v1 pool',  # noqa: E501
             counterparty=CPT_BALANCER_V1,
             address=string_to_evm_address('0x59A19D8c652FA0284f44113D0ff9aBa70bd46fB4'),
         ), EvmEvent(
             tx_hash=tx_hash,
             sequence_index=3,
             timestamp=timestamp,
             location=Location.ETHEREUM,
-            event_type=HistoryEventType.WITHDRAWAL,
-            event_subtype=HistoryEventSubType.REMOVE_ASSET,
-            asset=A_WETH,
-            balance=Balance(amount=FVal('0.010687148200906598')),
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.RETURN_WRAPPED,
+            asset=A_BPT,
+            balance=Balance(amount=FVal('0.042569019597126949')),
             location_label=user_address,
-            notes='Receive 0.010687148200906598 WETH after removing liquidity from a Balancer v1 pool',  # noqa: E501
+            notes='Return 0.042569019597126949 BPT to a Balancer v1 pool',
```

Comment:
So this seems wrong. The IN event comes first and the out later. rotki's events should always be OUT first, in afterwards